core/driver/sources/src/main/resources/reference.conf

```diff
@@ -161,34 +201,108 @@ probe {
       "https://cache-redirector.jetbrains.com/intellij-jbr/jbr-[major]-[platform]-x64-b[minor].tar.gz"
     ]
 
+    // The maximal number of retries that should be performed by resolvers when resolving a pattern to a resource.
     retries = 0
   }
 
   // `driver` is for `org.virtuslab.ideprobe.config.DriverConfig` - default values below
   driver {
+
+    // `driver.launch` contains settings that define how IDE should be started.
     launch {
+
+      // `driver.launch.command` is a string array transformed into the command used to launch the IDE (each string
+      // from the array is a single element of the command, spaces are inserted between the elements). By default,
+      // the driver uses the `idea` command, but you can specify a custom command by using this config.
```

Comment:
Should it then be `command = ["idea"]`?

Reply:
👍 In my opinion - yes. We can create an issue for that need. In fact, current solution allows both versions and treats them the same. But the `command` was an empty array before creating `reference.conf`, that's why it is what it is now. However, I think we can do a fast cleanup and set `command = ["idea"]` as the default one (with some small changes).
<img width="591" alt="image" src="https://user-images.githubusercontent.com/49597713/197879916-d47b28ea-5db6-4a05-ad49-eeb50db37d6d.png">

Reply:
Ok, +1